app/services/refresh_reporting_views.rb

```diff
@@ -74,15 +65,11 @@ def self.tz
   # LatestBloodPressuresPerPatientPerMonth should be refreshed before
   # LatestBloodPressuresPerPatientPerQuarter and LatestBloodPressuresPerPatient
   def refresh_v1
-    ActiveRecord::Base.transaction do
```

Comment:
The tx happens for each `refresh` call now for all matviews, so no need for an outer transaction.